src/Rendering/prePassRenderer.ts

```diff
@@ -257,6 +317,48 @@ export class PrePassRenderer {
         }
     }
 
+    private _updateGeometryBufferLayout() {
+        if (this._geometryBuffer) {
+            this._geometryBuffer.resetLayout();
+
+            const attachments = this._defaultAttachments.slice();
+            const gl = this._scene.getEngine()._gl;
+            attachments[0] = gl.NONE;

```

Comment:
I'm not sure it's allowed to access the gl layer in such high level classes... @sebavan ?

Maybe we need a function in the engine to retrieve this value?